english/eat-rainbows/right/right.md

```diff
@@ -2,4 +2,14 @@ You turned right and slammed into another customer carrying a birthday cake.
 
 Ughh, cake from Safeway with lard icing!
 
-Worst. Day. Ever.
\ No newline at end of file
+Worst. Day. Ever.
+Then you went directly to the bathroom trying to wash off the stains, however,
+you hesitate whether you should use the soap or only hand-wash with tap water. What will you choose?
+
+
+
+
+[Soap](english/eat-rainbows/soap/soap.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.